de.dashup.application/src/test/java/de.dashup.test/steps/LoginRegisterStepdefs.java

```diff
@@ -106,7 +106,7 @@ public void userCanCloseDashupAndOpenItAgainWithoutBeingLoggedOut() throws IOExc
         //we assert that the cookie value is correct and the same as the token on the DB
         HashMap<String, Object> whereParams = new HashMap<>();
         whereParams.put("user_id", "1");
-        Assertions.assertEquals(database.get(Database.Table.USERS_TOKENS, whereParams).getJSONObject(0)
+        Assertions.assertEquals(database.get(Database.Table.TOKENS, whereParams).getJSONObject(0)
```

Comment:
Why changing the name of the table? Since this table represents the user's tokens, I think this name fits very well.

Reply:
Can a token refer to anything else but a user? The combination of two table names should only be used for n:m intermediate tables.